src/components/MySpace/MySpace.test.tsx

```diff
@@ -96,8 +96,6 @@ describe('My Space Component', () => {
         })
       ).toBeInTheDocument()
 
-      // console.log(await screen.findAllByRole('listitem'))
-
```

Comment:
i ran across this and took it out to clean up, if someone wants it left in, let me know!